daemon/oci_linux.go

```diff
@@ -1052,7 +1052,7 @@ func (daemon *Daemon) createSpec(ctx context.Context, c *container.Container) (r
 		snapshotKey = c.ID
 	}
 
-	return &s, coci.ApplyOpts(context.Background(), nil, &containers.Container{
+	return &s, coci.ApplyOpts(ctx, nil, &containers.Container{
```

Comment:
This looks like a fix-up for the first commit as well

Reply:
Done